asv_bench/benchmarks/groupby.py

```diff
@@ -6,10 +6,15 @@
 import numpy as np
 
 from pandas import (
-    Categorical, DataFrame, MultiIndex, Series, TimeGrouper, Timestamp,
+    Categorical, DataFrame, MultiIndex, Series, Timestamp,
     date_range, period_range)
 import pandas.util.testing as tm
 
+try:
+    from pandas import TimeGrouper
```

Comment:
You can remove TimeGrouper altogether (see #26477)

Reply:
Ok.